src/diffusers_interpret/data.py

```diff
@@ -184,5 +210,32 @@ class AttributionAlgorithm(ExplicitEnum):
 
 @dataclass
 class AttributionMethods:
-    tokens_attribution_method: Union[str, AttributionAlgorithm] = AttributionAlgorithm.GRAD_X_INPUT
-    pixels_attribution_method: Optional[Union[str, AttributionAlgorithm]] = AttributionAlgorithm.MAX_GRAD
\ No newline at end of file
+    '''
+    Args:
+        tokens_attribution_method (`AttributionAlgorithm`)

```

Comment:
Do we need to put `AttributionAlgorithm`, seems like we shouldn't
```suggestion
        tokens_attribution_method (AttributionAlgorithm)

```
Same comment for the rest.
Example in the docs:
![image](https://user-images.githubusercontent.com/17574157/192600711-c8862534-1226-4a5f-b4dc-7be3255bed54.png)